regen/feature.pl

```diff
@@ -302,6 +302,8 @@ sub longest {
     print $h "#define FEATURE_BUNDLE_$key	", $count++, "\n";
 }
 
+# I think that the test for (CURRENT_HINTS & HINT_LOCALIZE_HH) can actually
```

Comment:
"I" in a comment in committed code is rather unclear. Either put your name, or add a signature line to the comment to explain who you are who are thinking this.

Reply:
Ill fix that.

Reply:
Fixed.